hash/h88_cass.xml

```diff
@@ -0,0 +1,161 @@
+<?xml version="1.0"?>
+<!DOCTYPE softwarelist SYSTEM "softwarelist.dtd">
+<!--
+license:CC0-1.0
+-->
+<softwarelist name="h88_cass" description="Heath H88 cassettes">
+
+	<software name="bug8_02_05_00">
+		<description>BUG-8 02.05.00</description>
+		<year>1978</year>
+		<publisher>Heath Corp.</publisher>
+		<info name="alt_title" value="Heath/Wintek Terminal Debugger"/>
+		<info name="developer" value="Heath Corp./Wintek Corp."/>
+		<info name="distributor" value="Heath Corp."/>
+		<info name="release" value="197806xx"/>
+		<info name="version" value="02.05.00"/>
+
+		<part name="cass" interface="h88_cass_player">
+			<dataarea name="cass" size="5027">
+				<rom name="bug8_02_05_00.h8t" size="5027" crc="6013dd00" sha1="805f89fee7ade4cf9d2401b111006634d9239d92"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="bug8_02_06_00">
+		<description>BUG-8 02.06.00</description>
+		<year>1978</year>
+		<publisher>Heath Corp.</publisher>
+		<info name="alt_title" value="Heath/Wintek Terminal Debugger"/>
+		<info name="developer" value="Heath Corp./Wintek Corp."/>
+		<info name="distributor" value="Heath Corp."/>
+		<info name="release" value="197806xx"/>
+		<info name="version" value="02.06.00"/>
+
+		<part name="cass" interface="h88_cass_player">
+			<dataarea name="cass" size="5143">
+				<rom name="bug8_02_06_00.h8t" size="5143" crc="63ab0c02" sha1="29c0b0242dca535eebc911288df40f34813cf08b"/>
+			</dataarea>
+		</part>
+	</software>
```

Comment:
Are these different versions of the same software?  If they are, the earlier version(s) should be marked as clones of the latest version.